### PR TITLE
feat(split-button-dropdown-text): add toggle text to split button

### LINKED ISSUE #2161: Add text to Split Button
Need ability to include a text string inside of a split button to support Bulk Selector design. See #1089 for design details.

### PR DESCRIPTION
**Pull request by mattnolting**
fixes #2161

### CODE REVIEW

File: src/patternfly/components/Dropdown/docs/code.md [hunk 1]
```diff
@@ -6,41 +6,42 @@ The dropdown menu can contain either links or buttons, depending on the expected
...
 
 | Class | Applied | Outcome |
 | -- | -- | -- |
-| `.pf-c-dropdown` | `<div>` | Defines the parent wrapper of the dropdown |
-| `.pf-c-dropdown__toggle` | `<button>` | Defines the dropdown toggle |
-| `.pf-c-dropdown__toggle-icon` | `<i>` | Defines the dropdown toggle icon |
+| `.pf-c-dropdown` | `<div>` | Defines the parent wrapper of the dropdown. |
+| `.pf-c-dropdown__toggle` | `<button>` | Defines the dropdown toggle. |
+| `.pf-c-dropdown__toggle-icon` | `<i>` | Defines the dropdown toggle icon. |
 | `.pf-c-dropdown__toggle-text` | `<span>` | Defines the dropdown toggle text. ** Required when text is present, adds truncation ** |
```
Comment (srambach) [inline src/patternfly/components/Dropdown/docs/code.md]: This renders with ** instead of being bold (extra spaces)

**Comment by srambach** [comment]
LGTM unless on the documentation you want `.`s after the "Required" text fragments. None of them are really full sentences, but it looks funny not to have a `.` when all the others do.

**Comment by mattnolting** [comment]
@srambach @mcoker updated

**Pull request merged**